components/Subscribe.tsx

```diff
@@ -72,8 +72,8 @@ export default function Subscribe({ campaign }: { campaign: string }) {
             </Form.Group>
           </Col>
         </Row>
-        <Row noGutters>
-          <Col style={{ padding: 0 }}>
+        <Row className="p-2">
```

Comment:
I think overall padding/margin was altered for Row/Col, so this just makes things look fine.